src/methods/sep24/pollDepositUntilComplete.ts

```diff
@@ -0,0 +1,115 @@
+import { log } from "helpers/log";
+
+export const pollDepositUntilComplete = async ({
+  popup,
+  transactionId,
+  token,
+  sep24TransferServerUrl,
+  trustAssetCallback,
+}: {
+  popup: any;
+  transactionId: string;
+  token: string;
+  sep24TransferServerUrl: string;
+  trustAssetCallback: () => Promise<string>;
+}) => {
+  let currentStatus = "incomplete";
+  let trustedAssetAdded;
+
+  const transactionUrl = new URL(
+    `${sep24TransferServerUrl}/transaction?id=${transactionId}`,
+  );
+  log.instruction({
+    title: `Polling for updates: ${transactionUrl.toString()}`,
+  });
+
+  while (!popup.closed && !["completed", "error"].includes(currentStatus)) {
```

Comment:
Lets add `pending_external` to this list, since external systems could take a long time to process a transaction.

Reply:
Fixed in https://github.com/stellar/stellar-demo-wallet/pull/116